client/hotrod-client/src/test/java/org/infinispan/client/hotrod/HitsAwareCacheManagersTest.java

```diff
@@ -188,26 +190,33 @@ protected void destroy() {
     * @author Mircea.Markus@jboss.com
     * @since 4.1
     */
-   public static class HitCountInterceptor extends BaseAsyncInterceptor{
+   public static class HitCountInterceptor extends BaseAsyncInterceptor {
       private static final Log log = LogFactory.getLog(HitCountInterceptor.class);
 
-      private final AtomicInteger invocationCount = new AtomicInteger(0);
+      private final AtomicInteger localSiteInvocationCount = new AtomicInteger(0);
+      private final AtomicInteger backupSiteInvocationCount = new AtomicInteger(0);
 
       @Override
-      public Object visitCommand(InvocationContext ctx, VisitableCommand command) throws Throwable {
+      public Object visitCommand(InvocationContext ctx, VisitableCommand command) {
          if (ctx.isOriginLocal()) {
-            int count = invocationCount.incrementAndGet();
-            log.infof("Hit %d for %s", count, command);
+            if ((command instanceof AbstractDataCommand) && ((AbstractDataCommand)command).getFlags().contains(Flag.SKIP_XSITE_BACKUP)) {
```

Comment:
```suggestion
            if ((command instanceof AbstractDataCommand) && ((AbstractDataCommand)command).hasAnyFlag(FlagBitSets.SKIP_XSITE_BACKUP)) {
```
(needs an import as well)